test/e2e/specs/OIDC_spec.js

```diff
@@ -57,6 +57,20 @@ describe('OIDC flows', function() {
       expect(oidcApp.getIdTokenUser()).toBe('Saml Jackson');
     });
 
+    it('throws form error if auth client returns with OAuth error', function() {
+      setup({
+        baseUrl: '{{{WIDGET_TEST_SERVER}}}',
+        clientId: 'rW47c465c1wc3MKzHznu',
+        redirectUri: 'http://localhost:3000/done',
+        authParams: {
+          responseType: 'id_token',
+          scope: ['openid', 'email', 'profile', 'address', 'phone']
+        }
+      });
+      primaryAuth.loginToForm('{{{WIDGET_BASIC_USER_5}}}', '{{{WIDGET_BASIC_PASSWORD_5}}}');
+      expect(primaryAuth.$dataSe('o-form-error-conainer').isPresent()).toBe(true);
```

Comment:
Hmm, so we need to add this user to Travis to actually test this scenario - I'm guessing that currently the error we're checking for is actually just the "You tried to submit a form without any credentials" type of error message. 

Also, as a side note, it'd be nice to add the $dataSe stuff to the page object rather than doing it here. Ideally at this level the logic would be on the level of "expect(primaryAuth.hasInvalidUserErrorMessage())" or something like that.